.github/scripts/review_issue.py

```
@@ -0,0 +1,348 @@
+import os
+from typing import List, Dict, Any
+import regex as re
+from github import Github
+from github.Issue import Issue
+from github.Repository import Repository
+# from qdrant_client import QdrantClient
+# from qdrant_client.models import PointStruct
+import openai
+
+if not os.getenv('GITHUB_ACTIONS'):
+    from dotenv import load_dotenv
+    load_dotenv()
+
+EMBEDDING_MODEL = "text-embedding-3-small"
+COLLECTION_NAME = "issue_collection"
+GPT_MODEL = "gpt-4o"
+MAX_RESULTS = 3
+
+class Config:
+    def __init__(self):
+        print("設定の初期化を開始します...")
+        self.github_token = os.getenv("GITHUB_TOKEN")
+        if self.github_token is None:
+            print("GITHUB_TOKENが見つかりません ...")
+        else:
+            print("GITHUB_TOKENからトークンを正常に取得しました。")
+        
+        # self.qd_api_key = os.getenv("QD_API_KEY")
+        # print("QD_API_KEYの状態:", "取得済み" if self.qd_api_key else "見つかりません")
+        
+        # self.qd_url = os.getenv("QD_URL")
+        # print("QD_URLの状態:", "取得済み" if self.qd_url else "見つかりません")
+        
+        self.github_repo = os.getenv("GITHUB_REPOSITORY")
+        print("GITHUB_REPOSITORYの状態:", "取得済み" if self.github_repo else "見つかりません")
+        
+        self.issue_number = os.getenv("GITHUB_EVENT_ISSUE_NUMBER")
+        if self.issue_number:
+            self.issue_number = int(self.issue_number)
+            print(f"GITHUB_EVENT_ISSUE_NUMBER: {self.issue_number}")
+        else:
+            print("GITHUB_EVENT_ISSUE_NUMBERが見つかりません")
+        print("設定の初期化が完了しました。")
+
+class GithubHandler:
+    def __init__(self, config: Config):
+        self.github = Github(config.github_token)
+        self.repo = self.github.get_repo(config.github_repo)
+        self.issue = self.repo.get_issue(config.issue_number)
+
+    def create_labels(self):
```

ラベルは現状すでに作られているのでこのメソッドは不要そう...？
カラーコードも既存のものと違っているので、再構築されると混乱を招きそうに思いました。
既に存在する場合は無視、のロジックがどこに入っているのかわからなかったのですが `self.repo.create_label`  (Repository.create_label)の内部で実装されてるんですかね？

---

確かに・・GithubHandler.create_labels の try except pass で既存のラベルがある場合は省略されるものの、不要なコードなので消します。

---

消しました！